.travis.yml

```diff
@@ -26,8 +26,8 @@ deploy:
   on:
     tags: true
     all_branches: true
-    rvm: 1.9.3
-    rvm: 2.0
```

Comment:
I see it was already removed in gemspec: https://github.com/sensu-plugins/sensu-plugins-mailgun/commit/e14928cad331bdf3e01acd9a9822954ac86388c3